src/File/FileHelper.php

```diff
@@ -43,7 +44,7 @@ public function absolutizePath(string $path): string
 				return $path;
 			}
 		}
-		if (str_starts_with($path, 'phar://')) {
+		if (str_contains($path, '://')) {
```

Comment:
AFAIK this change isn't covered by tests

Reply:
test added

Reply:
done